### DISCUSSION TITLE
Add Word-Wrapping to the Auto-Complete List

### DISCUSSION THREAD

**Discussion by RichNSD** [Acode-Foundation/Acode#646] [Ideas]
# Summary

Please add __Word Wrapping__ to the __Descriptions__ of items in the __Auto-Complete List__.

<br />

---

# Details

First, I want to point out how much I love __Acode's `.snippets` Syntax__. The fact that I don't have to write everything in a format compatible with `.json` is beautiful.

That said, I'm trying to build a series of __Snippet Packs__ that I intend to make available for anyone who wants to use them. Because I intend to distribute these snippets, I'm (trying) to take advantage of the `Description` field as much as possible.

Unfortunately, not much information can fit into the description before the text is inevitably cut off as it extends beyond the bounds of the __Auto-Complete List__.

### Snippet Code

```
snippet if-test [Logical Statement] | Test the expression according to -[test]
	if ${1:!}[[ -${2:Test Option} "${3:Path or Variable}" ]]; then
		${4:#statements}
	fi
	}${0}
```

### How it Renders

![autocomplete-word-wrapping-01](https://user-images.githubusercontent.com/100762874/217625265-584c53f6-42d4-42d8-b37f-60ec316aaa21.jpg)

<br />

## My Goal

I would honestly like to add *more* information to the snippet above, however there's no point in doing so if what little I've added so far is already being cut off. Here's an example of what the snippet would look like if I was able to add more detail:

### Potential Snippet Code
```
snippet if-test [Logical Statement] | Test the '[[ $expression ]]' to -[option]. Use the 'help test' command for a list of options.
	if ${1:!}[[ -${2:Test Option} "${3:Path or Variable}" ]]; then
		${4:#statements}
	fi
	}${0}
```

Adding these extra details will, among other things, make __learning how to code__ much easier for beginners- particularly those who don't have access to IDEs like __Visual Studio Code__.

Let me know what you think.

**Comment by deadlyjack** [comment]
Thanks for suggestion, I'll try to add word wrap in auto-complete prompt if possible.
